gensrc/thrift/AgentService.thrift

```diff
@@ -233,10 +234,17 @@ struct TRecoverTabletReq {
     4: optional Types.TVersionHash version_hash // Deprecated
 }
 
+enum TTabletMetaType {
+    PARTITIONID,
```

Comment:
I think this name is ambiguous, Is INMEMORY can not has partitionid?

Reply:
I think that is no ambiguous. One meta has one type, which is clear. All meta could be combine.